### PR TITLE
Switch example staging and prod workspace URLs based on cloud provider

### PR DESCRIPTION
**Pull request by shreyas-goenka**
## Changes
This PR parameterises the example host URLs displayed in prompts based on the cloud provider that a user selects.

Note, changes in this PR required a `min_cli_version` of v0.208.1. Do not merge until a correct minimum CLI version is set in the template schema.

## Tests
 Tested Manually.

### CODE REVIEW

**Comment by shreyas-goenka** [comment]
Prompts when aws is selected:
```
shreyas.goenka@THW32HFW6T playground % cli bundle init ~/mlops-stack
Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md. 

Project Name [my-mlops-project]: 

Root directory name. Use a name different from the project name if you intend to use monorepo [my-mlops-project]: 

Select cloud. 
Choose from azure, aws [azure]: aws

Select CICD platform. 
Choose from github_actions, github_actions_for_github_enterprise_servers, azure_devops [github_actions]: 

URL of staging Databricks workspace, used to run CI tests on PRs and preview config changes before they're deployed to production.
Example: https://your-staging-workspace.cloud.databricks.com
: 

URL of production Databricks workspace.
Example: https://your-prod-workspace.cloud.databricks.com
: ^C
```

**Comment by shreyas-goenka** [comment]
Prompts when azure is selected:
```
shreyas.goenka@THW32HFW6T playground % cli bundle init ~/mlops-stack
Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md. 

Project Name [my-mlops-project]: 

Root directory name. Use a name different from the project name if you intend to use monorepo [my-mlops-project]: 

Select cloud. 
Choose from azure, aws [azure]: azure

Select CICD platform. 
Choose from github_actions, github_actions_for_github_enterprise_servers, azure_devops [github_actions]: 

URL of staging Databricks workspace, used to run CI tests on PRs and preview config changes before they're deployed to production.
Example: https://adb-xxxx.xx.azuredatabricks.net
: 

URL of production Databricks workspace.
Example: https://adb-xxxx.xx.azuredatabricks.net
: ^C
```

**Review by arpitjasa-db — APPROVED** [review]
Looks great thanks @shreyas-goenka !

Follow-up question though, since this looks similar to conditional prompts, I wonder if we can use them as such? For example, if `Models in UC` is selected yes, then we could change `Include MLflow Recipes` to have an incompatible message like `MLflow recipes is not supported with Models in UC`.

File: databricks_template_schema.json [hunk 1]
```diff
@@ -28,13 +28,13 @@
       "order": 5,
       "type": "string",
       "default": "",
```
Comment (mingyu89) [inline databricks_template_schema.json:30]: Does the `default` field support the `if-else` branch? Currently we check if the value is "" and set to the default value based on cloud.
We don't need the if-else in the description if we support `default` field.

**Review by mingyu89 — APPROVED** [review]
Nice!
This change depends on the min version check PR.

File: databricks_template_schema.json (hunk 1 above)
Comment (shreyas-goenka) [reply databricks_template_schema.json:30]: Good point. `if-else` is indeed supported by the default field. Changed to use it instead.

**Comment by shreyas-goenka** [comment]
Output logs, after the default field is parameterised:
```
shreyas.goenka@THW32HFW6T playground % cli bundle init ~/mlops-stack
Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md. 

Project Name [my-mlops-project]: 

Root directory name. Use a name different from the project name if you intend to use monorepo [my-mlops-project]: 

Select cloud. 
Choose from azure, aws [azure]: 

Select CICD platform. 
Choose from github_actions, github_actions_for_github_enterprise_servers, azure_devops [github_actions]: 

URL of staging Databricks workspace, used to run CI tests on PRs and preview config changes before they're deployed to production. Default [https://adb-xxxx.xx.azuredatabricks.net]: 

URL of production Databricks workspace. Default [https://adb-xxxx.xx.azuredatabricks.net]: ^C
```
```
shreyas.goenka@THW32HFW6T playground % cli bundle init ~/mlops-stack
Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md. 

Project Name [my-mlops-project]: 

Root directory name. Use a name different from the project name if you intend to use monorepo [my-mlops-project]: 

Select cloud. 
Choose from azure, aws [azure]: aws

Select CICD platform. 
Choose from github_actions, github_actions_for_github_enterprise_servers, azure_devops [github_actions]: 

URL of staging Databricks workspace, used to run CI tests on PRs and preview config changes before they're deployed to production. Default [https://your-staging-workspace.cloud.databricks.com]: 

URL of production Databricks workspace. Default [https://your-prod-workspace.cloud.databricks.com]: ^C
```

**Review by arpitjasa-db — APPROVED** [review]
LGTM thanks @shreyas-goenka !

**Pull request merged**